src/System.Private.CoreLib/shared/System/Buffers/StandardFormat.cs

```diff
@@ -67,7 +67,7 @@ public StandardFormat(char symbol, byte precision = NoPrecision)
         public static implicit operator StandardFormat(char symbol) => new StandardFormat(symbol);
 
         /// <summary>
-        /// Converts a ReadOnlySpan<char> into a StandardFormat
+        /// Converts a <see cref="ReadOnlySpan{Char}"/> into a StandardFormat
```

Comment:
💡 This is not the reference it looks like. It should be this:

```suggestion
        /// Converts a <see cref="ReadOnlySpan{T}"/> into a StandardFormat
```

Reply:
> This is not the reference it looks like

What do you mean?